### PR TITLE
Add cairo_font_type_t

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/sys/issues/116

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
Maybe all enums need be exported under cairo names?

File: cairo-sys-rs/src/lib.rs [hunk 1]
```diff
@@ -58,6 +58,7 @@ pub type CairoSvgUnit = c_int;
 pub type CairoTextClusterFlags = c_int;
 
 pub type cairo_antialias_t = CairoAntialias;
+pub type cairo_font_type_t = CairoFontType;
```
Comment (sdroege) [inline cairo-sys-rs/src/lib.rs]: Maybe in -sys we should only have the names as in C, and the CamelCase enum types only in the non-sys crate? That's more consistent with everything else.

Comment (EPashkin) [reply cairo-sys-rs/src/lib.rs]: Agree.
@GuillaumeGomez What you think?

Comment (GuillaumeGomez) [reply cairo-sys-rs/src/lib.rs]: I agree as well.

**Comment by EPashkin** [comment]
Ok then I do changes here

**Comment by EPashkin** [comment]
Updated
Also found that we have two cairo_fill_rule_t :wink:

**Comment by GuillaumeGomez** [comment]
Thanks a lot!

**Pull request merged**